metaflow/plugins/kfp/kfp_s3_sensor.py

```diff
@@ -7,28 +7,68 @@
 from email.policy import default
 import click
 
+import os
 import pathlib
 from typing import Dict
+import boto3
+import botocore
+import base64
+import json
+import marshal
+import time
+from urllib.parse import urlparse
+
+from typing import Tuple
+
+
+def get_elapsed_time_s3_bucket_and_key(
+    flow_name: str, kfp_run_id: str
+) -> Tuple[str, str]:
+    s3_path = os.path.join(
+        os.getenv("METAFLOW_DATASTORE_SYSROOT_S3"),
+        flow_name,
+        kfp_run_id,
+        "_s3_sensor",
+    )
+    s3_path_parsed = urlparse(s3_path)
+    bucket, key = s3_path_parsed.netloc, s3_path_parsed.path.lstrip("/")
+    return bucket, key
```

Comment:
this function is contructing it rather than getting it from somewhere.

it: bucket, key

Reply:
Good point, `get_elapsed_time_s3_bucket_and_key` -> `construct_elapsed_time_s3_bucket_and_key`?